src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -18,11 +18,22 @@ static ucs_config_field_t uct_rdmacm_md_config_table[] = {
 
   {"ADDR_RESOLVE_TIMEOUT", "500ms",
    "Time to wait for address resolution to complete",
-    ucs_offsetof(uct_rdmacm_md_config_t, addr_resolve_timeout), UCS_CONFIG_TYPE_TIME},
+   ucs_offsetof(uct_rdmacm_md_config_t, addr_resolve_timeout), UCS_CONFIG_TYPE_TIME},
 
   {NULL}
 };
 
+static ucs_config_field_t uct_rdmacm_cm_config_table[] = {
+    {"", "", NULL,
+     ucs_offsetof(uct_rdmacm_cm_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_cm_config_table)},
+
+    {"SOURCE_ADDRESS", "",
+     "Source address used in rdma_resolve_addr",
```

Comment:
please add the expected format in the comment.

Reply:
is it ok now?
```
#
# Source IPv4 or IPv6 address used in rdma_resolve_addr or empty (equivalent to NULL)
#
# syntax:    string
#
UCX_RDMA_CM_SOURCE_ADDRESS=
```